### ISSUE TITLE
Unable to pass Single value as argument.

### ISSUE THREAD

**Issue by Guevara-chan** [AtsushiSuzuki/node-clr#9]
While working with CSCore library recently, I discovered interesting bug: sometimes `Double` datatype, which is default marshalling type for JS floats, could simply not be accepted by some functions, resulting in constant errors like this:
```
ArgumentException: Невозможно преобразовать объект типа "System.Double" к типу "System.Single".
   в System.RuntimeType.TryChangeType(Object value, Binder binder, CultureInfo culture, Boolean needsSpecialCast)
   в System.RuntimeType.CheckValue(Object value, Binder binder, CultureInfo culture, BindingFlags invokeAttr)
   в System.Reflection.MethodBase.CheckArguments(Object[] parameters, Binder binder, BindingFlags invokeAttr, CultureInfo culture, Signature sig)
   в System.Reflection.RuntimeMethodInfo.InvokeArgumentsCheck(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   в System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   в CLRBinder.InvokeMethod(Local<v8::Value>* , Type type, String name, Object target, Local<v8::Array> args) в c:\users\guevara-chan\appdata\roaming\npm\node_modules\clr\src\clrbinder.cc:строка 82
   в CLRBinder.InvokeMethod(Local<v8::Value>* , Local<v8::Value> typeName, Local<v8::Value> name, Local<v8::Value> target, Local<v8::Value> args) в c:\users\guevara-chan\appdata\roaming\npm\node_modules\clr\src\clrbinder.cc:строка 62
   в CLR.InvokeMethod(FunctionCallbackInfo<v8::Value>* info) в c:\users\guevara-chan\appdata\roaming\npm\node_modules\clr\src\node-clr.cc:строка 322
    at WasapiOut.Object.defineProperty.set [as Volume] (C:\Users\Guevara-chan\AppData\Roaming\npm\node_modules\clr\lib\clr.js:237:42)
```

Could something be ever done here ? Like, for example, forcing `node-clr` to pass some values as given type ?

**Comment by AtsushiSuzuki** [comment]
Seems that numeric type conversion is not properly working.

**Comment by Guevara-chan** [comment]
Seems so. Is there any workarounds at given time ?

**Comment by AtsushiSuzuki** [comment]
fixed in 0.0.18
